### ISSUE TITLE
Output unit number of Akita

### ISSUE THREAD

**Issue by frostinassiky** [calico/basenji#81]
Thanks for the amazing job!

According to the Akita Tutorial, we need to specify model parameters json to have only two targets.
```
params_file   = './params.json'
with open(params_file) as params_file:
    params_tutorial = json.load(params_file)   
params_tutorial['model']['head_hic'][-1]['units'] =2
```
ref: https://github.com/calico/basenji/blob/master/manuscripts/akita/tutorial.ipynb

However, the original parameters json have 5 targets. What are the extra 3 targets?

**Comment by frostinassiky** [comment]
BTW, does the learning curve look good for a production Akita training? Early stopping happens on the 35-th epoch.

![akita_production](https://user-images.githubusercontent.com/20427401/115586318-248ebc80-a2d5-11eb-9003-547e452bfda9.png)

**Comment by davek44** [comment]
The tutorial chooses two of the datasets arbitrarily to demonstrate the code. The primary model that we studied in the paper was trained on the five target datasets described here: https://github.com/calico/basenji/blob/master/manuscripts/akita/data/targets.txt

Yes, those training curves look good. I'm guessing your showing the training set statistics since I don't think early stopping would have chosen to stop if that were the validation set statistics.

**Comment by frostinassiky** [comment]
Hi @davek44 Thanks for your response! Do you have direct links for the three datasets: GM12878, IMR90, and HCT116? 

- they are not in the google storage `https://storage.googleapis.com/basenji_hic/tutorials/coolers/GM12878_inSitu_MboI_all.hg38.2048.cool` 
- they are not in the repo
`../../data`

**Comment by davek44** [comment]
If you want all of the datasets, you should consider using the preprocessed dataset into TFRecords, which you can acquire with this script: https://github.com/calico/basenji/blob/master/manuscripts/akita/get_data.sh

If you want all of the cooler files, I added them to the cloud bucket here: https://console.cloud.google.com/storage/browser/basenji_hic/1m/data/coolers
